tests/generation/test_utils.py

```diff
@@ -123,9 +123,13 @@ def _get_logits_processor_and_kwargs(
         process_kwargs = {
             "min_length": input_length + 1 if max_length is None else max_length - 1,
             "bad_words_ids": [[1, 0]],
-            "no_repeat_ngram_size": 2,
             "repetition_penalty": 1.2,
+            "remove_invalid_values": True,
         }
+        # NoRepeatNGramLogitsProcessor + forced tokens may result in no valid continuations
```

Comment:
This was the cause of the flakiness in the BART test (and very likely in other models as well!)